omod/src/main/webapp/resources/partials/daterangepicker.html

```diff
@@ -1,13 +1,14 @@
-<div class="date-range-picker-div" class="inlineBox">
-    <p> {{headermessage}} </p>
-    <span class="angular-datepicker">
-        <input class="appoinmentrangepicker" type="text" is-open="startDateOptions.opened" ng-model="startDate" min="{{startDateMin}}" max="endDate" show-weeks="false" datepicker-popup="dd MMM yyyy" readonly/>
-        <i class="icon-calendar small add-on" ng-click="startDateOptions.open($event)" ></i>
-        <a class="date-range-picker-clear-link add-on" ng-click="startDateOptions.clear($event)">{{ clearLinkText }}</a>
+<div class = "col-xs-5">
+  <div class="date-range-picker-div">
```

Comment:
you still reference a class that you deleted out

Reply:
Referencing this class has no effect on this because , it has been called in other different classes, what can be the better way.